test/test_helper.rb

```diff
@@ -5,6 +5,7 @@
 require 'esp_sdk'
 require 'minitest/autorun'
 require 'minitest/reporters'
+require 'mocha/mini_test'
```

Comment:
Oh, I thought you commented on a different repo, so I was looking at the wrong code. The mini test one was still needed, I'll see if the other require is needed.